### PR TITLE
Fix failure message of e2e test about kubectl

### LINKED ISSUE #68279: Wrong message of e2e test "apply set/view last-applied" about kubectl
**Is this a BUG REPORT or FEATURE REQUEST?**:

/kind bug

**What happened**:

https://github.com/kubernetes/kubernetes/blob/master/test/e2e/kubectl/kubectl.go#L805-L807

At e2e test for kubectl above, success scenario is `replicas` key is not contained in result of `view-last-applied`.
But test failure message is `Missing "replicas": 2 in kubectl view-last-applied`, in spite of `replicas` key is contained.

**What you expected to happen**:

When test for absence of `replicas` key failed, correct message is `Presenting "replicas": 2 in kubectl view-last-applied`.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
```
Client Version: version.Info{Major:"1", Minor:"13+", GitVersion:"v1.13.0-alpha.0.1057+8f906fefae84be-dirty", GitCommit:"8f906fefae84bebf5fe1ba60a4f2c8b5f1a6d775", GitTreeState:"dirty", BuildDate:"2018-09-05T07:30:12Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"13+", GitVersion:"v1.13.0-alpha.0.1057+8f906fefae84be-dirty", GitCommit:"8f906fefae84bebf5fe1ba60a4f2c8b5f1a6d775", GitTreeState:"dirty", BuildDate:"2018-09-05T07:30:12Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"linux/amd64"}
```
- Cloud provider or hardware configuration:none, hack/local-up-cluster.sh
- OS (e.g. from /etc/os-release):16.04.5 LTS (Xenial Xerus)
- Kernel (e.g. `uname -a`):4.4.0-124-generic
- Install tools:hack/local-up-cluster.sh
- Others:

### PR DESCRIPTION
**Pull request by atoato88**
**What this PR does / why we need it**:

At e2e test for "apply set/view last-applied",
failure message is `Missing "replicas": 2 in kubectl
view-last-applied`, in spite of `replicas` key is contained.
This PR changes `Missing` to `Presenting`.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #68279 

**Special notes for your reviewer**:

**Release note**:

```release-note
None
```

### CODE REVIEW

File: test/e2e/kubectl/kubectl.go [hunk 1]
```diff
@@ -799,11 +799,11 @@ metadata:
 			By("apply file doesn't have replicas")
 			framework.RunKubectlOrDieInput(deployment2Yaml, "apply", "set-last-applied", "-f", "-", nsFlag)
 
-			By("check last-applied has been updated, annotations doesn't replicas")
+			By("check last-applied has been updated, annotations doesn't have replicas")
 			output = framework.RunKubectlOrDieInput(deployment1Yaml, "apply", "view-last-applied", "-f", "-", nsFlag, "-o", "json")
 			requiredString = "\"replicas\": 2"
 			if strings.Contains(output, requiredString) {
-				framework.Failf("Missing %s in kubectl view-last-applied", requiredString)
+				framework.Failf("Presenting %s in kubectl view-last-applied", requiredString)
```
Comment (oomichi) [inline test/e2e/kubectl/kubectl.go]: This seems just copy&paste mistake from line 796, so LGTM

**Review by oomichi — COMMENTED** [review]
/lgtm

**Comment by atoato88** [comment]
/assign @mengqiy
for approval

**Pull request merged**